lib/matplotlib/cbook.py

```diff
@@ -1734,7 +1735,7 @@ def recursive_remove(path):
                 os.removedirs(fname)
             else:
                 os.remove(fname)
-        #os.removedirs(path)
+        # os.removedirs(path)
```

Comment:
I just changed the comment part according to the pep8 style. Nothing is changed within the function.